Improve specification for Coq source files

Why does the build system want to use the emacs backup file?

```
/Library/Developer/CommandLineTools/usr/bin/make -C cava
make[1]: *** No rule to make target `Cava/Core/.#Netlist.v', needed by `_CoqProject'.  Stop.
make: *** [cava] Error 2
```

--- a/cava/Makefile
+++ b/cava/Makefile
@@ -42,7 +42,7 @@ haskell:	coq FixupHaskell
 Makefile.coq:	_CoqProject
 		coq_makefile -f _CoqProject -o Makefile.coq
 
-_CoqProject:    $(shell find . -name '*.v' | sort)
+_CoqProject:    $(shell find . -name '[^\.]*.v' | sort)
 		@echo "INSTALLDEFAULTROOT = Cava" > _CoqProject
 		@echo "-R Cava Cava" >> _CoqProject
 		@echo "-R ../third_party/coq-ext-lib/theories ExtLib" >> _CoqProject

